page/ajax/working-with-jsonp.md

```diff
@@ -30,5 +30,9 @@ $.ajax({
 	}
 });
 ```
+If you'd like to access community tables you have to add `env: "http://datatables.org/alltables.env"` to the data object otherwise you would get an error message with `No definition found for Table`.
```

Comment:
We don't really have a strong markdown style guide, but could you add a blank line right after the end of the code snippet? It's consistent with the other markdown.